.opencode/skills/react-native-best-practices/references/bundle-library-size.md

```
@@ -0,0 +1,177 @@
+
+### Shows
+
+- **Minified size**: Raw JS size
+- **Minified + Gzipped**: Network transfer size
+- **Download time**: Estimated on various connections
+- **Dependencies**: What else gets pulled in
+- **Composition**: Breakdown by dependency
+
+### Example Analysis
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)